### PR TITLE
Post: Senseless SOQL

### PR DESCRIPTION
**Pull request by scottbcovert**

### CODE REVIEW

File: _drafts/senseless-soql.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+[Scott Covert](https://www.linkedin.com/in/scottbcovert/)
+
+<br/>
+
+<p align="center">
+  <img alt="" src="/images/2023-11-03-senseless-soql-doesnt-make-sense.gif" />
+</p>
+
+As a Salesforce professional you may be used to seeing quirky behavior due to the many nuances of the platform. But today we wanted to share with you some things that...
```
Comment (ndrewr) [inline _drafts/senseless-soql.md:19]: Good (and relatable) first sentence. To keep the second sentence more inline with the first, instead of "But ...", how about something like 

> Recently we came across some special SOQL behavior that we'd like to spotlight. To put it simply, the following does not. Make. Sense!

File: _drafts/senseless-soql.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+</sup>
+
+<p align="center">
+  <img alt="" src="/images/2023-11-03-senseless-soql-from-a-certain-point-of-view.jpeg" />
+</p>
+
+### Elementary Mathematics
+
+The next bit of senselessness first involves dusting off the old math book.
+Do you remember the Symmetric Property of Equality from your elementary mathematics? No? That's okay - we'll refresh your memory. This property states that
```
Comment (ndrewr) [inline _drafts/senseless-soql.md:49]: I'd shorten this to just:

>Recall the _Symmetric Property of Equality_ which states:

File: _drafts/senseless-soql.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+`If TabDefinition.DurableId = PermissionSetTabSetting.Name, then PermissionSetTabSetting.Name = TabDefinition.DurableId`
+
+However, as we now know, SOQL can be senseless.
+
+<p align="center">
+  <img alt="" src="/images/2023-11-03-senseless-soql-meme.jpeg" />
+</p>
+
+If you'd like to learn more about the Permissions Assistant tool we're building, or if you have your own SOQL stories to share, please [contact us](mailto:support@tython.co)!
```
Comment (ndrewr) [inline _drafts/senseless-soql.md:125]: Maybe we can tie the whole "making sense" angle back into the call-to-action, e.g. like

> If you have your own SOQL stories to share, or if you think the Permissions Assistant tool we are building can help restore some sense to your org's permissions quagmire ...

**Review by ndrewr — COMMENTED** [review]
Great idea for a blog post, left some minor feedback on a few sentences. 

I know Martin is testing the linked demo, but something that leapt out to me is that the layout might be broken. At least going by preview-mode in both VS Code and on github:
https://github.com/tythonco/tython-website/blob/011f599490b4715c002cdcc644251efab323f0eb/_drafts/senseless-soql.md

image

Maybe the log statements can be inside code block ticks?

Also, tangentially, this post makes me realize we really need to start a blogging series about Tython PA!
I'm going to create an issue on the internal board for that and self assign at least a first post in this series. I'll put a side-task in there to return to this post to edit in a link to that once its published 🔗

File: _drafts/senseless-soql.md (hunk 2 above)
Comment (scottbcovert) [reply _drafts/senseless-soql.md:49]: 👌

**Comment by scottbcovert** [comment]
> Great idea for a blog post, left some minor feedback on a few sentences.
> 
> I know Martin is testing the linked demo, but something that leapt out to me is that the layout might be broken. At least going by preview-mode in both VS Code and on github: https://github.com/tythonco/tython-website/blob/011f599490b4715c002cdcc644251efab323f0eb/_drafts/senseless-soql.md
> 
> image
> 
> Maybe the log statements can be inside code block ticks?
> 
> Also, tangentially, this post makes me realize we really need to start a blogging series about Tython PA! I'm going to create an issue on the internal board for that and self assign at least a first post in this series. I'll put a side-task in there to return to this post to edit in a link to that once its published 🔗
> 
> Edit: Tested on live dev server and it looks fine there 👍 Must be a VS Code rendering thing.

Even though it seemed GitHub Pages was okay with the markdown file I noticed that VS Code didn't like the unescaped ~ characters since that is usually used to represent strikethrough formatting in markdown.

**Comment by scottbcovert** [comment]
Thanks for the feedback @ndrewr - I applied all of your suggested changes and updated the logic to ignore any org namespaces per feedback from @martinglauber 

I'll go ahead and merge the PR so the post will be auto-published in the AM 👍
